src/components/fields/DescriptionField.js

```diff
@@ -10,18 +10,11 @@ function DescriptionField(props) {
     return <div />;
   }
 
-  const renderers = {};
-  if (window.renderCodeBlock) {
-    renderers.code = window.renderCodeBlock;
-  }
-
   return (
     <div id={id} className={pfx("field-description")}>
-      // TODO: children should be in child level
-      <ReactMarkdown
-        children={description.replace(/<br>/gi, "\n")}
-        renderers={{ ...renderers }}
-      />
+      <ContextConsumer>
```

Comment:
Pass `description.replace(/<br>/gi, "\n")` to `markdown Renderer` 

Constraints are not rendering properly.
![image](https://user-images.githubusercontent.com/17064396/134463703-9379ae5a-cd09-49c6-b05a-10e16b1e9bea.png)